index.html

```diff
@@ -0,0 +1,236 @@
+      <ul class="horizontal-menu px-4 mt-2">
+        <li>
+          <a href="index.html"><img class="navlogo " src="./capstoneimgs/logo.png" alt="world-music-festival-logo"></a>
+        </li>
+        <li class="sec-nav-item">
+          <a class="sec-nav-link px-4" href="about.html">About</a>
+        </li>
+        <li class="sec-nav-item">
+          <a class="sec-nav-link px-4" href="#">Program</a>
+        </li>
+        <li class="sec-nav-item">
+          <a class="sec-nav-link px-4" href="#">Performers</a>
+        </li>
+        <li class="sec-nav-item">
+          <a class="sec-nav-link px-4" href="#">Partners</a>
+        </li>
+        <li class="sec-nav-item-btn">
+          <button class="sec-nav-link-btn px-4">Campaign</button>
+        </li>
+      </ul>
+    </nav>
+
+    <div id="header">
+      <div id="header-nav-bar">
+        <nav id="burguer">
+          <div id="mobile-menu">
+            <img id="menu-burguer" src="./capstoneimgs/burguer.svg" alt="menu-button" class="menu-icon p-3">
+            <img id="close-menu" src="./capstoneimgs/x-button-icon-20.jpg" alt="close-menu" class="close-menu-icon p-3">
+          </div>
+          <div id="menu-container">
+            <ul class="menu">
+              <li class="nav-li1 p-2 w-100 d-flex justify-content-center text-danger">
+                <a href="about.html">About</a>
+              </li>
+              <li class="nav-li1 p-2 w-100 d-flex justify-content-center">
+                <a href="#program">Program</a>
+              </li>
+              <li class="nav-li1 p-2 w-100 d-flex justify-content-center">
+                <a href="#join">Join</a>
+              </li>
+              <li class="nav-li1 p-2 w-100 d-flex justify-content-center">
+                <a href="#sponsor">Sponsor</a>
+              </li>
+              <li class="nav-li1 p-2 w-100 d-flex justify-content-center">
+                <a href="#news">News</a>
+              </li>
+            </ul>
+          </div>
+        </nav>
+      </div>
+    </div>
+  </section>
+
+  <main>
+    <section class="hero">
+      <div class="intro sm d-flex-column">
+        <p class="hero-hello">"Hello! World Music Fans!"</p>
+        <h1 class="hero-title">INTI RAYMI MUSIC FESTIVAL 2023</h1>
+      </div>
+    </section>
+    <section id="artists-lineup-cont">
+      <div class="artists-lineup sm d-flex row justify-content-start">
+        <p class="line1">TAWA // TETRÁFONO ENSAMBLE // CHANGUITAS (Arg.)</p>
+        <p class="line2">TAMBORES Y OTROS DEMONIOS SAMY // GRUPO SENDERO // ALEX ALVEAR</p>
+        <p class="line3">IRAIZ OVIEDO Y SU MISTURA // MARIA TEJADA // TALLER LA BOLA // JATARI</p>
+        <p class="line4">WAÑUKTA TONIC // KARINA CLAVIJO // JAYAC // DANIEL MANCERO // PITEKUS</p>
+      </div>
+
+      <div class="date-venue">
+        <p class="date mx-4">2023.06.22(THU)</p>
+        <p class="venue mx-4">@Ciudad Mitad del Mundo, Central Plaza</p>
+      </div>
+    </section>
```

Comment:
- [ ] For the mobile version of your hero section it would be great if you can fix the size of the elements as they are way too large compared with the ones of the design:

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/226498195-0c604194-2b36-4b0b-9f92-41b9087dee2c.png) | ![image](https://user-images.githubusercontent.com/77069011/226498150-ef44520b-6e76-45e0-ae38-c869e91e5d65.png) |

- [ ] Also, your "artists-lineup" section should be designed as a text box to match the given design! remember that one of the main goals of this project is that you follow as closely as possible an existing design, please consider fixing this! 🤓 🔧

Reply:
Sorry about the broken image, I have fixed it and also aligned the items as requested 👍